packages/arb-avm-cpp/data_storage/include/data_storage/value/machine.hpp

```diff
@@ -41,8 +42,9 @@ struct MachineStateKeys {
     uint256_t arb_gas_remaining;
     CodePointRef pc{0, 0};
     CodePointStub err_pc{{0, 0}, getErrCodePoint()};
-    uint256_t staged_message_hash;
-    uint256_t total_messages_consumed;
+    uint256_t messages_fully_processed;
+    uint256_t inbox_accumulator;
```

Comment:
Similar name update as suggested in MachineState here I think

Reply:
👍